sphinx/source/docs/dev_guide/building.rst

```diff
@@ -75,42 +83,34 @@ the complete Bokeh library (see :ref:`devguide_python_setup`).
 
 However, if you are using BokehJS as a standalone JavaScript library, then
 the instructions below describe the process to iteratively build BokehJS with
-Grunt for development purposes.
-
-Building BokehJS with Grunt
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Gulp for development purposes.
 
-If you want to be able to run the commands below, you can either use
-``bokehjs/node_modules/.bin/grunt`` or globally install 
-the grunt command line interface::
+Building BokehJS with Gulp
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    $ npm install -g grunt-cli
-
-There are three main Grunt commands for development (to be executed from the
-``bokehjs`` subdirectory):
+Below are the main Gulp commands for development (to be executed from
+the ``bokehjs`` subdirectory). To run these commands, you can either
+use ``bokehjs/node_modules/.bin/gulp`` or globally install the Gulp
+command line interface:
 
 .. code-block:: sh
 
-    $ grunt deploy
+    npm install -g gulp
```

Comment:
Actually, this should be modified to tell it the user to run:

```console
conda install -c javascript gulp
```

That will get them the correct version that's maintained in-house.  You could add a "or install gulp via npm" as an option for people who don't have conda, but that's the best option.